sdks/builds/wasm.mk

```diff
@@ -38,25 +38,31 @@ $(TOP)/sdks/builds/toolchains/emsdk:
 .stamp-wasm-runtime-toolchain: .stamp-wasm-toolchain
 	touch $@
 
-.stamp-wasm-runtime-configure: $(TOP)/configure
-	mkdir -p $(TOP)/sdks/builds/wasm-runtime
-	cd $(TOP)/sdks/builds/wasm-runtime && source $(TOP)/sdks/builds/toolchains/emsdk/emsdk_env.sh && CFLAGS="-Os -g" emconfigure $(TOP)/configure $(WASM_RUNTIME_AC_VARS) $(WASM_RUNTIME_CONFIGURE_FLAGS)
+.stamp-wasm-runtime-$(CONFIGURATION)-configure: $(TOP)/configure
```

Comment:
Add the following to fix https://github.com/mono/mono/pull/10691/files#diff-77ef5cf8324c86c1370f3550e8eeffd2R145:
```
.PHONY: .stamp-wasm-runtime-configure
.stamp-wasm-runtime-configure: .stamp-wasm-runtime-$(CONFIGURATION)-configure
```

This duplicates the logic at https://github.com/mono/mono/blob/a4d0e665e190e551b043558d7c1d26bf5bcf705a/sdks/builds/runtime.mk#L70

Reply:
Ok, added